src/components/ParametersForm.jsx

```diff
@@ -0,0 +1,68 @@
+import {
+  FormControl, 
+  FormLabel, 
+  NumberInput, 
+  NumberInputField, 
+  NumberInputStepper, 
+  NumberIncrementStepper, 
+  NumberDecrementStepper,
+  Select,
+  Button,
+} from '@chakra-ui/react';
+
+
+export default function ParametersForm() {
+  return (
+    <>  
+      <FormControl>
+        <FormLabel>
+          <NumberInput 
```

Comment:
Maybe you can split the components even more so you would have your own custom `Button`, `NumberInput`, `Select`, etc.